### PR TITLE
Disable news fragments check

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
- Disable the check for the news fragment bits.
- Add a pull request template to question the pr's author if the changes need a fragment.
- Update the ci to only check the news fragments if there was an edit on the folder `newsfragments`

### CODE REVIEW

File: .github/actions/paths-filter/action.yml [hunk 1]
```diff
@@ -20,6 +20,9 @@ outputs:
   client-common:
     value: ${{ steps.changes.outputs.client-common }}
     description: True if a files in the ionic-client part was edited.
+  news-fragments:
+    value: ${{ steps.changes.outputs.news-fragments }}
```
Comment (touilleMan) [inline .github/actions/paths-filter/action.yml:24]: should stay consistent with the name `newsfragments` without dash

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -109,12 +112,7 @@ jobs:
       - name: Check Newsfragment
         if: |
           startsWith(github.ref, 'refs/pull/')
-          && !(
-            startsWith(github.head_ref, 'dependabot')
-            || startsWith(github.head_ref, 'yolo')
-            || startsWith(github.head_ref, 'release')
-            || startsWith(github.head_ref, 'revert')
-            )
+          && steps.changes.outputs.news-fragments
```
Comment (touilleMan) [inline .github/workflows/ci.yml:115]: So we use the check_newsfragments.py to check the provided newsfragement has the right format ?

**Review by touilleMan — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (FirelightFlagboy) [reply .github/workflows/ci.yml:115]: yes

File: .github/actions/paths-filter/action.yml (hunk 1 above)
Comment (FirelightFlagboy) [reply .github/actions/paths-filter/action.yml]: done

**Review by touilleMan — APPROVED** [review]

**Pull request merged**